File: app/src/main/java/com/spiderbiggen/randomchampionselector/model/Lane.java
package com.spiderbiggen.randomchampionselector.model;

/**
 *
 * @author Stefan Breetveld
 */
public enum Lane {
    DUO_SUPPORT("Support"),
    JUNGLE("Jungle"),
    TOP("Top"),
    MIDDLE("Mid"),
    DUO_CARRY("Bot");

    private String name;

    Lane(String name) {
        this.name = name;
    }

    @Override
    public String toString() {
        return name;
    }
}

File: app/src/main/java/com/spiderbiggen/randomchampionselector/model/Lane.kt
package com.spiderbiggen.randomchampionselector.model

/**
 * In game role description.
 *
 * @author Stefan Breetveld
 */
enum class Lane constructor(val description: String) {
    DUO_SUPPORT("Support"),
    JUNGLE("Jungle"),
    TOP("Top"),
    MIDDLE("Mid"),
    DUO_CARRY("Bot");

    override fun toString(): String {
        return description
    }
}
